services/121-service/src/registration/registrations.service.ts

```diff
@@ -1120,7 +1120,7 @@ export class RegistrationsService {
       if (error instanceof IntersolveVisaApiError) {
         throw new HttpException(
           `${IntersolveVisa121ErrorText.reissueCard} - ${error.message}`,
-          HttpStatus.INTERNAL_SERVER_ERROR,
```

Comment:
I don't think it's nice to throw an HttpException with code INTERNAL_SERVER_ERROR because we do not get an email from it, which is confusing as we do get one on new Error() 

So I think we should either throw a new Error or a HttpException with 400 or 404.

In this case we sort of take into account that this error can occur, namely when a user changes an address property or a phonenumber to something that is not allowed by intersolve. Therefore I think this error is expected so I used a 400 here and not new Error()

Reply:
Create a Task to look at alerting for 500 errors.